README.md

```diff
@@ -24,16 +24,19 @@ How to use
 - Import the motion sensor classes in your HTML (see [`<script type="module">` browser support](https://caniuse.com/#feat=es6-module) or use a [polyfill](https://github.com/ModuleLoader/es-module-loader) to load this polyfill!):
 ```
 <script type="module">
+// Import the classes you need.
 import {
-  Sensor,
   Accelerometer,
   LinearAccelerationSensor,
   Gyroscope,
   GravitySensor,
   RelativeOrientationSensor,
   AbsoluteOrientationSensor
 } from './src/motion-sensors.js';
+
+// And they're ready for use!
+sensor = new AbsoluteOrientationSensor();
```

Comment:
You said "you need" but you imported much more than you actually needed :-)

Reply:
Fixed :-)